src/_pytest/main.py

```diff
@@ -636,6 +626,18 @@ def _recurse(self, dirpath):
         ihook.pytest_collect_directory(path=dirpath, parent=self)
         return True
 
+    if six.PY2:
+
+        @staticmethod
+        def _visit_filter(f):
```

Comment:
i recall this can be stored as constant using `methodcaller('check', file=True, notext="pyc")`

Reply:
Not sure if I understand correctly, but there is no "notext" in py.

Reply:
there is a ext checker method and the not prefix of the kwarg triggers inversion ^^
its horrendous in a funny way

Reply:
Would rather keep it like it is now (i.e. only move it), but feel free to push an update, or patch here.

Reply:
@RonnyPfannschmidt 
What do you plan?  Either post a patch/push a fixup (feel free to squash), or approve it, please.

Reply:
yikes, thanks for the call out, i missed that bit